backend/src/main/scala/cromwell/backend/standard/StandardInitializationActor.scala

```diff
@@ -0,0 +1,67 @@
+package cromwell.backend.standard
+
+import akka.actor.ActorRef
+import cromwell.backend.validation.RuntimeAttributesDefault
+import cromwell.backend.{BackendConfigurationDescriptor, BackendWorkflowDescriptor, BackendWorkflowInitializationActor}
+import cromwell.core.WorkflowOptions
+import wdl4s.TaskCall
+import wdl4s.values.WdlValue
+
+import scala.concurrent.Future
+import scala.util.Try
+
+trait StandardInitializationActorParams {
+  val workflowDescriptor: BackendWorkflowDescriptor
+  val calls: Set[TaskCall]
+  val serviceRegistryActor: ActorRef
+  val configurationDescriptor: BackendConfigurationDescriptor
+}
+
+case class DefaultInitializationActorParams
+(
+  workflowDescriptor: BackendWorkflowDescriptor,
+  calls: Set[TaskCall],
+  serviceRegistryActor: ActorRef,
+  configurationDescriptor: BackendConfigurationDescriptor
+) extends StandardInitializationActorParams
+
+trait StandardInitializationActor extends BackendWorkflowInitializationActor {
+  /** Standard set of parameters passed to the backend. */
+  def standardParams: StandardInitializationActorParams
+
+  override lazy val serviceRegistryActor: ActorRef = standardParams.serviceRegistryActor
+
+  override lazy val calls: Set[TaskCall] = standardParams.calls
+
+  def runtimeAttributesBuilder: StandardValidatedRuntimeAttributesBuilder =
+    StandardValidatedRuntimeAttributesBuilder.default
+
+  override protected def runtimeAttributeValidators: Map[String, (Option[WdlValue]) => Boolean] = {
+    runtimeAttributesBuilder.validatorMap
+  }
+
+  override protected def coerceDefaultRuntimeAttributes(options: WorkflowOptions): Try[Map[String, WdlValue]] = {
+    RuntimeAttributesDefault.workflowOptionsDefault(options, runtimeAttributesBuilder.coercionMap)
+  }
+
+  override def validate(): Future[Unit] = {
+    Future.fromTry(Try {
```

Comment:
Whoops. I messed up above, swapping the "I prefer:" and "Instead of:".

As the actor `receive` only processes one mailbox message at a time, by design, I can understand if one wants to break up execution and not block an actor's mailbox. I'm just not a fan of using scala futures to do that, and would prefer to see child akka actors.